component/metrics/scrape/scrape.go

```diff
@@ -128,7 +128,7 @@ func (c *Component) Update(args component.Arguments) error {
 	defer c.mut.Unlock()
 	c.args = newArgs
 
-	c.appendable = fa.FlowAppendable(newArgs.ForwardTo)
```

Comment:
Just want to point this out: this was incorrect behavior :) 

This assigns a new value to c.appendable (i.e., a new pointer to memory containing a slice), while the scraper continues using the previous value of c.appendable (i.e., a different pointer to memory containing a different slice).

Reply:
i.e., for the update to work correctly the value of c.appendable _must_ stay the same, and instead the underlying memory c.appendable points to changes.